AnnService/inc/Core/Common/WorkSpace.h

```diff
@@ -67,7 +67,9 @@ namespace SPTAG
 
             void Init(SizeType size)
             {
+                m_poolSize = (int)pow(2, log2(size) + 1) - 1;
```

Comment:
it should be at least (int)pow(2, (int)(log2(size) + 1)) - 1;
but I suggest to use simple loop here. a pow function may have precision issue.